code/game/objects/effects/decals/cleanable.dm

```diff
@@ -6,7 +6,7 @@
 	var/bloodiness = 0 //0-100, amount of blood in this decal, used for making footprints and affecting the alpha of bloody footprints
 	var/mergeable_decal = TRUE //when two of these are on a same tile or do we need to merge them into just one?
 	var/beauty = 0
-	var/clean_type = CLEAN_TYPE_LIGHT_DECAL
+	var/clean_type = CLEAN_TYPE_LIGHT_DECAL //the type of cleaning required to clean the decal, CLEAN_TYPE_LIGHT_DECAL can be cleaned with mops and soap, CLEAN_TYPE_HARD_DECAL can be cleaned by soap, see __DEFINES/cleaning.dm for the others
```

Comment:
Use the autodoc format, as in putting the comment *before* the declaration, and using `/// ` instead of `//`.

Reply:
What about those other two I see right above?

Reply:
they're old, fix them if you are gracious enough